src/base.jl

```diff
@@ -1008,11 +976,19 @@ Return an Array of values for the requested time series parameters.
 function get_time_series_values(
     ::Type{T},
     component::Component,
-    initial_time::Dates.DateTime,
-    label::AbstractString,
-    horizon::Union{Nothing, Int} = nothing,
+    name::AbstractString;
+    start_time::Union{Nothing, Dates.DateTime} = nothing,
+    len::Union{Nothing, Int} = nothing,
+    count::Int = 1,
 ) where {T <: TimeSeriesData}
-    return IS.get_time_series_values(T, component, initial_time, label, horizon)
+    return IS.get_time_series_values(
```

Comment:
Same here, do we have this for Deterministic?

Reply:
Same as above